### ISSUE TITLE
[javascript] InaccurateNumericLiteral with number 259200000

### ISSUE THREAD

**Issue by tomasvts** [pmd/pmd#4716]
**Affects PMD Version:**
6.55.0 

**Rule:**
https://docs.pmd-code.org/latest/pmd_rules_ecmascript_errorprone.html#innaccuratenumericliteral

**Description:**
A seemingly normal constant is triggered by this rule. Tried different names for the constant also, but to no avail.

![image](https://github.com/pmd/pmd/assets/4351937/6f9ff546-d0fb-4e4c-aa8c-7dbf5842ff9a)

**Code Sample demonstrating the issue:**

```javascript
const USER_LIST_TRANSACTION_EXP_MS = 259200000;
```

**Expected outcome:**
PMD reports a violation at line ..., but that's wrong. That's a false positive.

**Running PMD through:** *Codacy*

![image](https://github.com/pmd/pmd/assets/4351937/a29d2e92-821e-4a46-80be-ea48bb52a85c)

**Comment by witaliy12** [comment]
Thanks you much
